Tests.NHibernate.Spatial.PostGis20/app.config

```diff
@@ -6,7 +6,7 @@
   </configSections>
 
   <connectionStrings>
-    <add name="Tests.NHibernate.Spatial.Properties.Settings.ConnectionString" connectionString="Server=localhost;Port=5432;Database=nhsp_test;User Id=nhsp_test;Password=nhsp_test;Encoding=UNICODE;" />
+    <add name="Tests.NHibernate.Spatial.Properties.Settings.ConnectionString" connectionString="Server=localhost;Port=5432;Database=nhsp_test;User Id=nhsp_test;Password=nhsp_test;" />
```

Comment:
Removed the `unicode` encoding as the test database is created with an [uft8 encoding](https://github.com/nhibernate/NHibernate.Spatial/blob/master/Tests.NHibernate.Spatial.PostGis20/nhsp_test.sql#L12), which is also the encoding that `Npgsql` uses by default.